### DISCUSSION TITLE
1.4.2

### DISCUSSION THREAD

**Discussion by villebro** [apache/superset#19402] [General]
Hello Community,

The Apache Superset team is pleased to announce that Superset 1.4.2 has just been released.

Apache Superset is a modern, enterprise-ready business intelligence web application.

The official source release:

https://www.apache.org/dist/superset/1.4.2

The Pypi package:

https://pypi.org/project/apache-superset/1.4.2

If you have any usage questions, or have problems when upgrading or
find any problems about enhancements included in this release, please
don't hesitate to let us know by sending feedback to this mailing
list.

---------------------------------------------------------------------
To unsubscribe, e-mail: general-unsubscribe@apache.org
For additional commands, e-mail: general-help@apache.org
--------------------------

<hr /><em>This discussion was created from the release <a href='https://github.com/apache/superset/releases/tag/1.4.2'>1.4.2</a>.</em>

**Comment by hugeboris** [comment]
I thought this version should include plugins code, but it's not, is it not latest version ?
